obsproject/obs-studio
# UI: Show link to release notes in Help menu

### Description
Dynamically shows a Release Notes link if the `OBS_VERSION` tag exists on GitHub.

![image](https://user-images.githubusercontent.com/941350/77239625-edad2980-6c30-11ea-897a-9d74494e348d.png)

### Motivation and Context
Currently, users are only exposed to the release notes once - before they update. This provides a quick and easy link that doesn't require digging.

### How Has This Been Tested?
* Build with a valid tag. See the link, click it.
* Build with a non-existent tag or without an internet connection. Don't see the link.

### Types of changes
- New feature (non-breaking change which adds functionality)

### Checklist:
